cli-tests/src/upload.rs

```diff
@@ -232,6 +235,59 @@ async fn upload_bundle_using_bep() {
     println!("{assert}");
 }
 
+#[tokio::test(flavor = "multi_thread")]
+async fn upload_bundle_success_status_code() {
+    let temp_dir = tempdir().unwrap();
+    generate_mock_git_repo(&temp_dir);
+    let test_bep_path = get_test_file_path("test_fixtures/bep_retries");
+    let uri_fail = format!(
+        "file://{}",
+        get_test_file_path("../cli/test_fixtures/junit0_fail.xml")
```

Comment:
I don't love looking out of the crate, but this is certainly useful test data that is nontrivial to generate. The next best thing would probably be to make a common test data dir directory, but I didn't want to undertake that unilaterally here